thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/route.js

```diff
@@ -27,213 +20,44 @@ export default Route.extend(AuthenticatedRouteMixin, {
 
   model() {
     return hash({
-      applications: fetch('/thirdeye/entity/APPLICATION').then(checkStatus),
-      detectionAlertConfig: fetch('/detection/subscription-groups').then(checkStatus),
-      polishedDetectionYaml: fetch('/yaml/list').then(checkStatus)
+      subscriptionGroupConfig: fetch('/detection/subscription-groups').then(checkStatus),
```

Comment:
It shouldn't need to get all subscription groups anymore, is it?

Reply:
@jihaozh This is fetched so that we can add Application to the alerts for display in the UI.  If we can add Application as a field in the response from /alerts endpoint, I can get rid of this.  To see what I'm talking about, check enrichAlertResponseObject() function in yaml-tools.js

Reply:
Alternatively, I could remove the Application field from the display of Alerts in the Alerts Route.  Let me know what you think is the best approach.

Reply:
Thanks for pointing out. I'll add the application into the endpoint's response so that the overall architecture is clean.